form/unit_testing.rst

```diff
@@ -227,14 +227,15 @@ a good opportunity to use them::
 
     class TestedTypeTest extends TypeTestCase
     {
-        /**
-         * @dataProvider getValidTestData
-         */
+       
```

Comment:
See https://phpunit.de/manual/6.2/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers